scripts/game.js

```diff
@@ -0,0 +1,64 @@
+var rows = 50;
+var cols = 100;
+
+var playing = false;
+
+// define grid using 'new' operator to create an instance of the grid object
+var grid = new Array(rows);
+var nextGrid = new Array(rows);
+
+var timer;
+var reproductionTime = 100;
+
+
+
+function buildGrid() {
+  for (var i = 0; i < rows; i++) {
+    grid[i] = new Array(cols);
+    nextGrid[i] = new Array(cols);
+  }
+}
+function loadGrid() {
+    buildGrid();
+}
+
+// click handler for when user clicks cell on grid and turns cell "on" or "off"
+function clickHandler() {
+
+    // loop through array with class of "on" and set to "off" when clicked and vice versa
+
+
+// Create game board grid
+function createGrid() {
+  var gridBoard = document.getElementById("grid-container");
+  console.log("grid maybe?", gridBoard);
+  var grid = document.createElement("grid");
```

Comment:
When using createElement you must pass it a string that equates to an HTML tag. I do not believe "grid" is an HTML tag. I believe you meant to put "div" and then give it the class "grid"